.travis.yml

```diff
@@ -81,4 +81,4 @@ script:
   fi
 
 after_success:
-  - if [ $SUBMIT_CODECOV == 'true' ]; then codecov; fi;
+  #- if [ $SUBMIT_CODECOV == 'true' ]; then codecov; fi;
```

Comment:
This can be better avoided by applying this config file: https://github.com/NSLS-II/scientific-python-cookiecutter/blob/master/%7B%7B%20cookiecutter.repo_name%20%7D%7D/.codecov.yml. I understand the comments about the changed coverage annoyed you, right?

Reply:
Well, actually I wanted to temporarily disable `codecov`, because coverage in PyXRF is not good and `codecov` started to fail builds. I am thinking that once the coverage improves, I will enable `codecov` once the coverage gets above 40%.

Reply:
Sounds good. But that `.codecov.yml` is worth to add anyway, so that next time it does not "spam" the PRs 😃.